aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Build a .NET MAUI Blazor app
+author: guardrex
+description: Build a .NET MAUI Blazor app step-by-step.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/12/2022
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/tutorials/maui
+---
+# Build a .NET MAUI Blazor app
+
+This tutorial shows you how to build and modify a .NET MAUI Blazor app. You learn how to:
+
+> [!div class="checklist"]
+> * Create a .NET MAUI Blazor app project
+> * Run the project in the Windows Machine emulator
+> * Run the project in the Android emulator
+
+At the end of this tutorial, you'll have a working app.
+
+## Prerequisites
+
+* [Supported platforms (.NET MAUI documentation)](/dotnet/maui/supported-platforms)
+* [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/) with the following workloads:
+  * **Mobile development with .NET**
+  * **Universal Windows Platform development**
+* [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/): WebView2 is required on Windows when running a native app. When developing .NET MAUI Blazor apps and only running them in Visual Studio's emulators, WebView2 isn't required.
+
+## Installation
+
+Follow the guidance in [Installation (.NET MAUI documentation)](/dotnet/maui/get-started/installation).
+
+## Create a .NET MAUI Blazor app
+
+Launch Visual Studio 2022 17.1 Preview, and in the start window click **Create a new project** to create a new project:
+
+:::image type="content" source="maui/_static/new-solution.png" alt-text="New solution.":::
+
+In the **Create a new project** window, select **MAUI** in the **Project type** drop-down, select the **.NET MAUI App (Preview)** template, and click the **Next** button:
```

Comment:
For a new project, we should just tell users to use the .NET MAUI Blazor template, which already has the BlazorWebView set up. It would be great to also have instructions on how to add a BlazorWebView to an existing .NET MAUI app that wasn't originally setup with Blazor, but let's focus on the getting started flow with the .NET MAUI Blazor app project template first.


```suggestion
In the **Create a new project** window, select **MAUI** in the **Project type** drop-down, select the **.NET MAUI Blazor App (Preview)** template, and click the **Next** button:
```